docs/eventing/sinks/job-sink.md

```diff
@@ -0,0 +1,305 @@
+asynchronous jobs and tasks.
+
+`JobSink` supports the full
+Kubernetes [batch/v1 Job resource and features](https://kubernetes.io/docs/concepts/workloads/controllers/job/)
+and Kubernetes Job queuing systems like [Kueue](https://kueue.sigs.k8s.io/).
+
+## Prerequisites
+
+You must have access to a Kubernetes cluster
+with [Knative Eventing installed](../../install/yaml-install/eventing/install-eventing-with-yaml.md).
+
+## Usage
+
+When an event is sent to a `JobSink`, Eventing creates a `Job` and mounts the received event as
+JSON file at `/etc/jobsink-event/event`.
+
+1. Create a `JobSink`
+    ```yaml
+    apiVersion: sinks.knative.dev/v1alpha1
+    kind: JobSink
+    metadata:
+      name: job-sink-logger
+    spec:
+      job:
+        spec:
+          completions: 1
+          parallelism: 1
+          template:
+            spec:
+              restartPolicy: Never
+              containers:
+                - name: main
+                  image: docker.io/library/bash:5
+                  command: [ "cat" ]
+                  args:
+                    - "/etc/jobsink-event/event"
+    ```
+2. Apply the `JobSink` resource:
+    ```shell
+    kubectl apply -f <job-sink-file.yaml>
+    ```
+3. Verify `JobSink` is ready:
+    ```shell
+    kubectl get jobsinks.sinks.knative.dev
+    ```
+   Example output:
+    ```shell
+    NAME              URL                                                                          AGE   READY   REASON
+    job-sink-logger   http://job-sink.knative-eventing.svc.cluster.local/default/job-sink-logger   5s    True
+    ```
+4. Trigger a `JobSink`
+   ```shell
+   kubectl run curl --image=curlimages/curl --rm=true --restart=Never -ti -- -X POST -v \
+      -H "content-type: application/json"  \
+      -H "ce-specversion: 1.0" \
+      -H "ce-source: my/curl/command" \
+      -H "ce-type: my.demo.event" \
+      -H "ce-id: 123" \
+      -d '{"details":"JobSinkDemo"}' \
+      http://job-sink.knative-eventing.svc.cluster.local/default/job-sink-logger
+   ```
+5. Verify a `Job` is created and prints the event:
+   ```shell
+   kubectl logs job-sink-loggerszoi6-dqbtq
+   ```
+   Example output:
+    ```shell
+    {"specversion":"1.0","id":"123","source":"my/curl/command","type":"my.demo.event","datacontenttype":"application/json","data":{"details":"JobSinkDemo"}}
+    ```
+
+### JobSink idempotency
+
+`JobSink` will create a job for each _different_ received event.
+
+An event is uniquely identified by the combination of event `source` and `id` attributes.
+
+If an event with the same `source` and `id` attributes is received and a job is already present,
+another `Job` will _not_ be created.
+
+### Reading the event file
+
+You can read the file and deserialize it using any [CloudEvents](https://github.com/cloudevents)
+JSON deserializer.
+
+For example, the following snippet reads an event using the CloudEvents Go SDK and processes it.
```

Comment:
I think this section is a little confusing without some more context, because as far as I can tell we don't mention before this that the event is mounted as a file into the job

Reply:
It's the first sentence after `# Usage` at line 24

Reply:
Ah, I guess I missed that - my bad!